<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-04-15T10:48:34 IST" name="Default suite" finished-at="2024-04-15T10:48:45 IST" duration-ms="11234">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.Runner@6200f9cb]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2024-04-15T10:48:34 IST" name="Default test" finished-at="2024-04-15T10:48:45 IST" duration-ms="11234">
      <class name="TestRunner.Runner">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:TestRunner.Runner@6200f9cb]" started-at="2024-04-15T10:48:34 IST" name="setUpClass" finished-at="2024-04-15T10:48:35 IST" duration-ms="540" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@238ad8c]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.Runner@6200f9cb]" started-at="2024-04-15T10:48:35 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-04-15T10:48:42 IST" duration-ms="7490" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA["1 login with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["login functionalities"]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.Runner@6200f9cb]" started-at="2024-04-15T10:48:35 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-04-15T10:48:45 IST" duration-ms="10062" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["2 login with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["login functionalities"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[confirmed expected [Administration] but found [Dashboard]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: confirmed expected [Administration] but found [Dashboard]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at POM.pomodel.verify(pomodel.java:39)
at Step.stepFile.verify_login_as(stepFile.java:92)
at ✽.verify  as "Administration"(file:///C:/Users/abhil/git/TestDemo/TestDemo/src/test/resources/rrr.feature:21)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:TestRunner.Runner@6200f9cb]" started-at="2024-04-15T10:48:35 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-04-15T10:48:44 IST" duration-ms="9661" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["3 login with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["login functionalities"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[confirmed expected [Administration!] but found [Administration]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: confirmed expected [Administration!] but found [Administration]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at POM.pomodel.verify(pomodel.java:39)
at Step.stepFile.verify_login_as(stepFile.java:92)
at ✽.verify  as "Administration!"(file:///C:/Users/abhil/git/TestDemo/TestDemo/src/test/resources/rrr.feature:30)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:TestRunner.Runner@6200f9cb]" started-at="2024-04-15T10:48:45 IST" name="tearDownClass" finished-at="2024-04-15T10:48:45 IST" duration-ms="564" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- TestRunner.Runner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
